app/hydrator/templates/create/toppanel.html

```diff
@@ -1,5 +1,5 @@
 <!--
-  Copyright © 2015-2017 Cask Data, Inc.
```

Comment:
This should be kept as a range going up to the current year. I.e. `2015-2022`

Reply:
updated